interfaces/PA-App/src/app/shared/password-toggle-input/password-toggle-input.component.html

```diff
@@ -5,12 +5,17 @@
   <div style="display: flex; width: 100%;"
        class="ion-align-items-center"
   >
-    <ion-input [type]="inputType"
+    <ion-input #passwordInput
+               [type]="inputType"
                [name]="name"
                [(ngModel)]="ngModel"
                ngDefaultControl
                [disabled]="disabled"
                [autocomplete]="autocomplete"
+               minlength="4"
```

Comment:
The exact length should be 'configurable' from the component.ts-file, so that we can also use the same number inside the translations/message.

I'll add that in a later PR.